site/src/pages/WorkspaceBuildParametersPage/WorkspaceBuildParametersPageView.tsx

```diff
@@ -150,7 +151,9 @@ export const WorkspaceBuildParametersPageView: FC<
 
       <Maybe
         condition={Boolean(
-          props.templateParameters && props.templateParameters.length === 0,
+          !props.isLoading &&
+            props.templateParameters &&
```

Comment:
Usage could be nicer here if we defaulted to an empty array for the prop.

Reply:
Before introducing `isLoading` it was not possible, but I can retake a look.

Reply:
Usually, it is not a good idea to use an empty array in the FE because we usually use undefined to identify the "loading data" state. However, if this data is provided internally and not from an external source that depends on an effect like network calls, file reading, etc. Providing an empty array is ok 👍

Reply:
Also, we could introduce `isLoading` and set it as an empty array but I don't see what is the value on it if compared to just letting the value as undefined.